lighthouse-core/lib/i18n/i18n.js

```diff
@@ -202,12 +199,12 @@ function collectAllCustomElementsFromICU(icuElements, seenElementsById = new Map
  * Returns a copy of the `values` object, with the values formatted based on how
  * they will be used in the `icuMessage`, e.g. KB or milliseconds. The original
  * object is unchanged.
- * @param {string} icuMessage
  * @param {MessageFormat} messageFormatter
  * @param {Readonly<Record<string, string | number>>} values
+ * @param {string} icuMessage Used for clear error logging.
```

Comment:
should we rename this param now that `icuMessage` means `that object thing returned from a string we want localized`

Reply:
> should we rename this param now that `icuMessage` means `that object thing returned from a string we want localized`

I'll use `lhlMessage` since it's in that format? Or should it be `rawMessage` or..something?